service/fileUpload.js

```diff
@@ -0,0 +1,36 @@
+const path = require('path');
+const multer = require('multer')
+const fs = require('fs').promises;
+const uploadDir = path.join(process.cwd(), 'tmp')
+const storeImage = path.join(process.cwd(), 'public/avatars')
```

Comment:
tmp public i public/avatars nie istnieją w repo. To przez to że git przechowywuje tylko pliki i ich ścieżki. Masz 2 opcje albo użyć fs przy starcie servera sprawdzić czy pliki istnieją i jak nie to je utworzyć (najpierw public potem public/avatars bo samo public/avatars wywali ci błąd jeśli public nie istnieje) albo dodać w tych folderach plik .gitkeep i w tedy przez to że coś jest w tych folderach to zapiszą się na repo

Reply:
dzięki, dodałem pliki .gitkeep